src/elm/Page/Register.elm

```diff
@@ -476,16 +522,40 @@ type alias UpdateResult =
 
 
 type Msg
-    = UpdateForm FormInputMsg
-    | ValidateForm Form
-    | GotAccountAvailabilityResponse Bool
-    | KeysGenerated (Result Decode.Error AccountKeys)
-    | CompletedSignUp AccountKeys (Result (Graphql.Http.Error (Maybe SignUpResult)) (Maybe SignUpResult))
+    = ValidateForm FormType
+    | GotAccountAvailabilityResponse (Result Decode.Error Bool)
+    | AccountGenerated (Result Decode.Error AccountKeys)
     | AgreedToSave12Words Bool
     | DownloadPdf PdfData
     | PdfDownloaded
     | CopyToClipboard String
     | CopiedToClipboard
+    | CompletedLoadInvite (Result (Graphql.Http.Error (Maybe Invite)) (Maybe Invite))
+    | CompletedLoadCountry (Result (Graphql.Http.Error (Maybe Address.Country)) (Maybe Address.Country))
+    | AccountTypeSelected AccountType
+    | FormMsg EitherFormMsg
+    | DefaultFormMsg1 DefaultForm.Msg
```

Comment:
Looks like `DefaultFormMsg1` and `EitherFormMsg` are somehow similar in working with the Default form. Maybe we use `EitherFormMsg` only?